packages/open-api-gateway/src/project/smithy/components/sample-executable.ts

```diff
@@ -0,0 +1,38 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+import * as fs from "fs";
+import * as path from "path";
+import { Project, SampleFile, SampleFileOptions } from "projen";
+import { getFilePermissions } from "projen/lib/util";
+
+/**
+ * A sample file that is executable
+ */
+export class SampleExecutable extends SampleFile {
+  private readonly file: string;
```

Comment:
minor: rename to filePath

Reply:
I named it that first but jsii complained because I was overriding a subclass property haha :)